.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 /lib
 /server/static/
 *.log
+!.env
```

Comment:
I have it in my global .gitignore, it's always a good practice to ignore it I think.